lib/executor.js

```diff
@@ -1,23 +1,27 @@
-var EventEmitter = require('./emitter').EventEmitter;
-var _ = require('lodash');
-var util = require('util');
-var selfAddressed = require('self-addressed');
+const _ = require('lodash');
+const util = require('util');
+const selfAddressed = require('self-addressed');
+const EventEmitter = require('./emitter').EventEmitter;
+const logger = require('./logging');
 
 exports.TaskExec = TaskExec;
 
-function TaskExec() {
+function TaskExec({ loggerFields } = {}) {
     EventEmitter.call(this);
     this.errorCount = 0;
+    this.logger = logger.child(loggerFields || {});
```

Comment:
I think it's better to restrict log interface to text message only, in other words, calls like this `this.log.wan('Some message', {prop1: 'val1', prop2: 'val2'})` should ignore `{prop1: 'val1', prop2: 'val2'}` and log only `'Some message'` and add pnly props defined in `loggerFields`, not more.

Reply:
done

```
this.logger.info(); // check if info level is enabled
this.logger.info('test'); // log test
this.logger.info('test %s', 123); // bunyan will format message
this.logger.info({}); // first object argument will be implicitly converted to string 
```